src/Resources/Locales/en.json

```diff
@@ -1341,6 +1341,8 @@
         "unknownErrorTitle": "Unknown error",
         "corsErrorTitle": "CORS error",
         "corsErrorMessage": "Your storage account needs to have certain CORS properties enabled to be able to use 3D assets in your scenes.",
+        "partialCorsErrorTitle": "CORS update required",
+        "partialCorsErrorMessage": "Due to recent feature changes, there are some CORS rules which need to be updated on your storage account to be able to continue using 3D assets in your scenes.",
```

Comment:
I would remove the word Feature. "In order to keep using 3D scenes, there are some..."